src/bin/pgcopydb/ld_stream.c

```diff
@@ -100,29 +190,84 @@ stream_init_specs(StreamSpecs *specs,
 
 		case STREAM_PLUGIN_WAL2JSON:
 		{
+			/* Initialize empty strings for tables to add and filter */
+			char *commaSeperatedAddTables = { 0 };
+			char *commaSeperatedFilterTables = { 0 };
+
```

Comment:
I would suggest to declare the PQExpBuffer addTablesBuffer & PQExpBuffer filterTablesBuffer here and destroy it at the end of function or on error.
```suggestion
			PQExpBuffer addTablesBuffer = createPQExpBuffer();
			PQExpBuffer filterTablesBuffer = createPQExpBuffer();

```